### PR TITLE
Onboard card!

### PR DESCRIPTION
**Pull request by tobyab**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/**](https://site-px9b4i1y8.hackclub.dev/) |
|:-:|
|<a href="https://site-px9b4i1y8.hackclub.dev/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/7VmPthNgvqmlcq9DZJCmmnQdKhlTiGa3KEfczabi0lrp_G8PLUFqvxicrgxW7R0VLU_7RA2ADKS69IwmDFChBA.png">(view full size)</a> |



And 1 other route:
- [**/api/stars**](https://site-px9b4i1y8.hackclub.dev/api/stars)



Commit 136e517cadb3e75041457c43cd7af89265f862da (https://site-px9b4i1y8.hackclub.dev).

File: pages/api/stars.js [hunk 1]
```diff
@@ -1,26 +1,27 @@
-import { graphql } from '@octokit/graphql'
+import { graphql } from "@octokit/graphql";
```
Comment (maxwofford) [inline pages/api/stars.js]: Did you mean to reformat this page?

Comment (tobyab) [reply pages/api/stars.js]: Just fixed! My editor refuses to turn off it's auto-formatter for some reason.

**Comment by tobyab** [comment]
Am I all good to merge, @maxwofford?

**Comment by tobyab** [comment]
Thanks, mate!

**Pull request merged**